lib/ssl/src/ssl.erl

```diff
@@ -977,25 +996,25 @@ insecure to use, or plainly not relevant anymore.
   >
   > Using `{padding_check, false}` makes TLS vulnerable to the Poodle attack.
```

Comment:
While you are at it , this ought to be a warning.

Reply:
It already **is** a warning (line 976/995, not shown by default in the diff).

Reply:
Ok then, will look at the built documentation once it's built. Agree diffs get you confused sometimes :)